be/src/storage/rowset/rle_page.h

```diff
@@ -93,7 +93,11 @@ class RlePageBuilder final : public PageBuilder {
         for (int i = 0; i < count; ++i) {
             // note: vals is not guaranteed to be aligned for now, thus memcpy here
             CppType value;
-            memcpy(&value, &new_vals[i], SIZE_OF_TYPE);
+            if constexpr (std::is_same<CppType, bool>::value) {
+                value = (new_vals[i] != 0);
```

Comment:
```suggestion
                value = new_val[i];
```
It is not recommended to compare a boolean value with 0.